contracts/drafts/TokenVesting.sol

```diff
@@ -16,8 +16,8 @@ contract TokenVesting is Ownable {
   using SafeMath for uint256;
   using SafeERC20 for IERC20;
 
-  event Released(uint256 amount);
-  event Revoked();
+  event TokensReleased(address token, uint256 amount);
+  event TokensRevoked(address token);
```

Comment:
I still think this should be `TokenVestingRevoked`. I'm not sure if you are ignoring my comment because you disagree...

Reply:
Haha I don't, it's actually a great suggestion. I was just tired yesterday and misread your comment :grin: